dev/io.openliberty.org.eclipse.microprofile/faulttolerance.4.1.bnd

```diff
@@ -19,9 +19,9 @@ Export-Package: \
   org.eclipse.microprofile.faulttolerance.exceptions;version=1.0
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;4.0.2;EXACT}
+  @${repo;org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;4.1.0.RC1;EXACT}
 
 -maven-dependencies: \
-   dep1;groupId=org.eclipse.microprofile.fault-tolerance;artifactId=microprofile-fault-tolerance-api;version=4.0.2;scope=runtime
+   dep1;groupId=org.eclipse.microprofile.fault-tolerance;artifactId=microprofile-fault-tolerance-api;version=4.1-RC1;scope=runtime
```

Comment:
```suggestion
   dep1;groupId=org.eclipse.microprofile.fault-tolerance;artifactId=microprofile-fault-tolerance-api;version=4.1.RC1;scope=runtime
```

Reply:
I don't think this is right. This version, as I understand it, creates a dependency entry in `META-INF/maven/dev/io.openliberty.org.eclipse.microprofile.faulttolerance.4.1/pom.xml`.

I'm not quite sure why we do this, but I assume it's supposed to contain the coordinates for the upstream API jar. In this case the version needs to be `4.1-RC1` to match the maven central coordinates.

With the suggested change, the file contains this, which will not resolve to the upstream API jar:
```xml
    <dependency>
      <groupId>org.eclipse.microprofile.fault-tolerance</groupId>
      <artifactId>microprofile-fault-tolerance-api</artifactId>
      <version>4.1.RC1</version>
      <scope>runtime</scope>
    </dependency>
```

Reply:
That is what I thought as well, but that isn't what we were doing with the Jakarta ones.  We do this for tracking what OSS we pull into the product and have to do it because we are using sub bundles and nothing is on the buildpath to show what we are pulling in.  I will mark it resolved and approve the PR.  Will follow up with the Rest Client guys to update theirs as well.

Reply:
Huh, interesting, I didn't know that was what it was for. I'm interested to know what different thing we've done on the Jakarta APIs.